Plaid API Endpoints Collection.json

```diff
@@ -5641,7 +5676,7 @@
 							},
 							"response": [
 								{
-									"name": "Error response",
```

Comment:
I'm not sure why some of these changed? I simply imported the existing collection and then exported it after adding the new endpoint.

Reply:
(it might just be a re-ordering)